File: Design Patterns in Java/Chain of Responsibility Pattern/src/dev/chamalwr/handler/ContentHandler.java
package dev.chamalwr.handler;

import dev.chamalwr.model.Email;

/**This handler responsible of checking
 * email contains malicious content
 **/
public class ContentHandler extends EmailHandler{
    @Override
    public void checkMail(Email email) {
        String contentFormatted= email.getContent().toLowerCase();
        if (!contentFormatted.contains("malicious")){
            System.out.println("Email is clean! Delivered Successfully");
        }else{
            System.err.println("Email has malicious content");
        }
    }
}

File: Design Patterns in JavaScript/Chain of Responsibility Pattern/handler/ContentHandler.js
const EmailHandler = require("./EmailHandler");

module.exports = class ContentHandler extends EmailHandler{
    contentToAvoid = "malicious"

    checkMail(email){
        let formattedText = email.content.toString().toLowerCase()
        if (!formattedText.includes(this.contentToAvoid)){
            console.log("Email is safe!")
            if (this.successorHandler != null){
                this.successorHandler.checkMail(email)
            }
        }else{
            console.log("Email Contains Malicious Content")
        }
    }
}
